modules/social_features/social_core/src/Service/LayoutService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\social_core\Service;
+
+use Drupal\core\Entity\EntityInterface;
+use Drupal\layout_builder\LayoutEntityHelperTrait;
+
+/**
+ * Class LayoutService.
+ *
+ * @package Drupal\social_core\Service
+ */
+class LayoutService {
+
+  use LayoutEntityHelperTrait;
```

Comment:
@ronaldtebrake Unfortunately this breaks if layout_builder is not enabled because Drupal's autoloading is dependant on enabled modules.

An example of a similar scenario that broke is here: https://www.drupal.org/project/social/issues/3102067

Reply:
Sorry, I did not yet see that we only enabled this service when the layout_builder is enabled. However, that probably means we can remove the snippet on line 27.
```
if (!\Drupal::moduleHandler()->moduleExists('layout_builder')) {
      return FALSE;
    }
```

This will never run because `moduleExists('layout_builder')` will always be true if this service is registered :)

Reply:
Yeah was merily a double measurement to avoid any other things registering it in the wrong time and or place. Not sure if that ever happens but felt like a good better safe than sorry thingy.